consensus/polybft/state_sync_manager_test.go

```diff
@@ -142,6 +142,50 @@ func TestStateSyncManager_MessagePool_SenderIsNoValidator(t *testing.T) {
 	require.Error(t, err)
 }
 
+func TestStateSyncManager_MessagePool_InvalidEpoch(t *testing.T) {
```

Comment:
You can probably add to other tests as well if you want.
```suggestion
func TestStateSyncManager_MessagePool_InvalidEpoch(t *testing.T) {
     t.Parallel()
```

Reply:
fixed 91d52fe